### DISCUSSION TITLE
Water vapor in L58 and L93

### DISCUSSION THREAD

**Discussion by JulioTBacmeister** [NCAR/amwg_dev#277] [TF2: Tuning]
@dan800 @tilmes @cecilehannay @islasimpson  Let's discuss water vapor here.  Questions
Are all the runs from #259 through #271 double counting water??

**Comment by cecilehannay** [comment]
Yes. They were done with a tag that still included the H2O external forcing. 
We removed the H2O external forcing in cam6_3_111 but we have not done a run with this yet. 

I could do a new control if needed. 
If we do, it would be nice to decide if we have a consensus on how we want to set tau_0_ubc

**Reply by adamrher** [reply]
thanks for clarifying @tilmes @dan800. As you say, it's the 3D emissions of H2O that may be causing the problem, not the upper b.c.'s. Let's see what the new control looks like. 

I am not the right person to consult on tau_0_ubc.

**Reply by cecilehannay** [reply]
I just talked to @JulioTBacmeister and we will start a new LT control with 
- cam6_3_112 (this tag has the H2O external forcing removed) and 
```
use_gw_convect_dp = .true.
use_gw_front = .true.
tau_0_ubc = .true.
```
- we will also do the L32 (#274) based on this new control 

PS: The first run is the same thing as what @dan800 is suggesting in #276

**Comment by JulioTBacmeister** [comment]
![image](https://github.com/NCAR/amwg_dev/assets/9221710/b22ffbdb-4dd4-49f0-8f19-640998a54fb8)
 
On the positive side it does seem that LT and MT water vapor in the UTLS are close

**Reply by dan800** [reply]
#269 seems closest over the southern pole in the upper stratosphere - presumably the strong vortex is isolating the pole and preventing mixing in of moist air from mid-latitudes.
